Makefile

```diff
@@ -7,8 +7,13 @@ UNAME_S ?= $(shell uname -s)
 # flag for testing
 CGO_CFLAGS := -DTESTING
 
-lorawan: sx1302
+default: clean lorawan
```

Comment:
not make the module by default? or is this better for testing

Reply:
Yeah I feel like this is more useful for testing since the make module will usually be done with CI.

Reply:
gotcha def leave it with whatever is best for your workflow